flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionMySQLSink.java

```diff
@@ -66,7 +66,7 @@
      */
     public OrionMySQLSink() {
         super();
-        logger = CygnusLogger.getLogger(OrionHDFSSink.class);
+        logger = CygnusLogger.getLogger(OrionMySQLSink.class);
```

Comment:
Issue create for release/0.6.0 --> https://github.com/telefonicaid/fiware-connectors/issues/321

CHANGES_NEXT_RELEASE updated in https://github.com/telefonicaid/fiware-connectors/commit/758f0c512a5bc3a8674d3f724e9fb5bae22d8ae3